docs/runtime-contract.md

```diff
@@ -282,9 +275,8 @@ code. These settings apply to both `livenessProbe` and `readinessProbe`:
 - `initialDelaySeconds` set to 0
 - `periodSeconds` set to platform-specific value
 
-In order to enable scaling in response to load, developers SHOULD NOT set the
-`initialDelaySeconds` to a value greater than 0, and should aim to minimize
-container startup time (aka cold start time).
+Setting `initialDelaySeconds` to a value of 0 will enable scaling in response
```

Comment:
I like the removal of SHOULD NOT. Two nits:

(1) While line 275 explains that 0 is the default this line seems to encourage explicitly setting to 0.
(2) This explains what happens when it is set to 0, but does not explain what happens when it goes past zero or why a developer should prefer to keep it unset/0.

Here is what I suggest:
```
Setting `initialDelaySeconds` to a value greater than 0 impacts container startup time (aka cold start time) as a container will not serve traffic until the probe succeeds.
```

Reply:
Agree with @dgerd but s/will not serve traffic until the probe succeeds/cannot receive traffic until the probe has started/, I think.

Reply:
Done